source/partials/schedule/_schedule_table.slim

```diff
@@ -1,7 +1,8 @@
 .ScheduleTable
   - data.program.each_with_index do |item, index|
-    .ScheduleTable-day
-      h2.SectionHeading.small.blue.u-smallPushLeft-mobile = "day #{index + 1}"
+    .ScheduleTable-entry
+      = table_day(index) do
```

Comment:
this helper should be named something like `schedule_table_day`. In here, out of context, it's not explicit enough